### ISSUE TITLE
Suggestion: Add link to report issues

### ISSUE THREAD

**Issue by fawcett** [mapswipe/app-feedback#3]
Obviously, Mapswipe is designed to appeal to a very broad audience, with a wide range of technical skills.  At the same time, you will likely draw a fair number of geospatial and IT professionals.  When they spot a bug, or have a suggestion, it would be great if there was a link to the Github project page or another places to report issues.

It could be as simple as a link on the contacts page.

**Comment by WolfSteigi** [comment]
I agree with fawcett. I may be nice to have an easy to use channel to send suggestion and other issues.
Regards, 
Wolfgang
